src/components/TopBar/TopBarStyles.js

```diff
@@ -0,0 +1,58 @@
+import styled from 'styled-components';
+
+export const StyledTopBarWrapper = styled.nav`
+  position: absolute;
+  top: 0;
+  right: 0;
+  width: 100%;
+  height: 57px;
+`;
+
+export const StyledTopBar = styled.div`
+  width: 100%;
+  height: 56px;
+  padding: 16px;
+  background-color: #2a2a2a;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  box-shadow: 0px 4px 6px rgba(0, 0, 0, 0.12), 0px 2px 4px rgba(0, 0, 0, 0.14),
+    0px 1px 2px rgba(0, 0, 0, 0.2);
+`;
+
+export const StyledHeader = styled.header`
+  font-family: Roboto;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 19px;
+  padding-left: 53px;
+  line-height: 22px;
+  display: flex;
+  align-items: center;
```

Comment:
>The CSS align-items property sets the align-self value on all direct children as a group.

I don't see `StyledHeader` having any children.
It has only text inside
```
  <StyledHeader>NeuralNotes</StyledHeader>
```

So I believe `align-items: center` doesn't have any effect here.

Same for `display: flex`.

Reply:
Got it)